tests/unit/converters/test_tabular_converter.py

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
 #
 # SPDX-License-Identifier: MPL-2.0
-
+import json
```

Comment:
Two new imports in this file, but I don't see them being used

Reply:
Fair point thnx, I'll remove them. I added a unit test here, but then decided that this is not the right place to test it.